install/README.md

```diff
@@ -35,66 +35,71 @@ Once the CSM installation has completed, other product streams for the HPE Cray
+> [general install troubleshooting section](#troubleshooting-installation-problems) and
+> [troubleshooting section](../operations/README.md) will offer assistance.
+
+## Pre-Installation
 
-The topics in this chapter need to be done as part of an ordered procedure so are shown here with numbered topics.
+This section will guide the administrator through creating and setting up the
+Cray Pre-Install Toolkit.
 
-> **`NOTE`** If problems are encountered during the installation, some topics have their own [troubleshooting sections found in the operations index](../operations/README.md)
-sections, but there is also a general troubleshooting topic.
+Fresh-installations may start at the
+[boot installation environment](#2-boot-installation-environment) section, re-installations will
+have other steps to complete in the
+[preparing for a re-installation](#1-preparing-for-a-re-installation) section.
 
-# Preparing for a Re-installation
+### 1. Preparing for a Re-installation
 
 If one is reinstalling a system, the existing cluster needs to be wiped and powered down.
 
 See [Prepare Management Nodes](re-installation.md), and then come back and proceed
 to the [Pre-Installation](#pre-installation) guide.
 
-# Pre-Installation
-
 These steps walk the user through properly setting up a CRAY supercomputer for an installation.
 
 See [pre-installation](./pre-installation.md).
 
-## 1. Boot Installation Environment
+### 2. Boot Installation Environment
 
 See [boot installation environment](./pre-installation.md#1-boot-installation-environment).
 
-## 2. Import CSM Tarball
+### 3. Import CSM Tarball
 
 See [import CSM tarball](./pre-installation.md#2-import-csm-tarball).
 
-## 3. Create System Configuration
+### 4. Create System Configuration
 
 See [create system configuration](./pre-installation.md#3-create-system-configuration).
 
-## 4. Configure Management Network Switches
+### 5. Configure Management Network Switches
 
-At this point external connectivity has been established, and either bare-metal configs can be installed or new configs can be applied.
+At this point external connectivity has been established, and either bare-metal configurations can
+be installed or new/updated configurations can be applied.
 
 See [Management Network User Guide](../operations/network/management_network/README.md).
 
-## 5. Identify Inventory
+### 6. Identify Inventory
 
-At this point inventory can be identified. This is important for validating the SHCD. Specifically 
+At this point inventory can be identified. This is important for validating the SHCD. Specifically
 BMCs will be located, interrogated, and their node types identified.
 
-**This section is not yet complete and exists as a placeholder, pleaes move onto [Installation](#installation).**
+**This section is not yet complete and exists as a placeholder, please move onto [Installation](#installation).**
 
 **`TODO: Add page for identification.`**
```

Comment:
Ideally if we want placeholder reminders, we should just open Jira tickets for them.